src/demo/app/model/demo.model.ts

```diff
@@ -0,0 +1,41 @@
+import { Observable } from 'rxjs/Observable';
+import { BaseModel, RelatedMap, HalDoc, REQUIRED } from 'ngx-prx-styleguide';
+
+export class DemoModel extends BaseModel {
+  public foo: string;
+  public complete: boolean;
+  public count: number;
+  public color: string;
+  public script: string;
+  public arrival: Date;
+  public feedId: string;
```

Comment:
`feedId` still in demoModel setable + decode but no longer used in fancy field -- maybe for use later?